### PR TITLE
884 style changes

### LINKED ISSUE #884: UI style/spacing/copy changes
components:
- internal lines (between dropdown options) match the row separator lines in table component
- ~common container backTo links have more top spacing~ (Resolved by #897)

report:
- standardize width of Other Revenue input boxes
- use ReportingPeriodFormatter formatted dates for report sub-header (currently, date range is manually formatted)

roster: 
-  ~update sub-header language for single-site roster view of multi-site org
current is misleading, b/c the site-specific X/30 does not accurately represent org-wide utilization. Instead, only use the "ratio" style in single-site org or unfiltered multi-site org view. In filtered single-site, update subheaders to:~
```
X in full time CDC space(s)
```
covered by #893

### PR DESCRIPTION
**Pull request by mmazanec22**
Closes #884

### CODE REVIEW

File: src/Hedwig/ClientApp/src/components/Header/_index.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@
 		}
 	}
 	a:focus .usa-logo__text {
-		@extend [href]:focus;
```
Comment (cailyncodes) [inline src/Hedwig/ClientApp/src/components/Header/_index.scss:17]: Interested in what this change does

File: src/Hedwig/ClientApp/src/containers/Enrollment/Detail/__snapshots__/EnrollmentDetail.test.tsx.snap [hunk 2]
```diff
@@ -246,7 +246,7 @@ exports[`EnrollmentDetail matches snapshot 1`] = `
             class="usa-hint text-italic"
           >
              
-            on March 1st, 2019
```
Comment (cailyncodes) [inline src/Hedwig/ClientApp/src/containers/Enrollment/Detail/__snapshots__/EnrollmentDetail.test.tsx.snap:249]: Looks like a regression ?

File: src/Hedwig/ClientApp/src/containers/Reports/ReportDetail/ReportDetail.tsx [hunk 3]
```diff
@@ -63,8 +63,8 @@ export default function ReportDetail() {
 				</div>
 				<p className="usa-intro">
 					{report.organization.name} <br />
-					{dateFormatter(report.reportingPeriod.periodStart)}–
-					{dateFormatter(report.reportingPeriod.periodEnd)}
+					{dateFormatter(report.reportingPeriod.periodStart, false)}–
```
Comment (jlhogan) [inline src/Hedwig/ClientApp/src/containers/Reports/ReportDetail/ReportDetail.tsx:66]: should this use reportingPeriodFormatter with extended format?

File: src/Hedwig/ClientApp/src/utils/dateFormatter.ts [hunk 4]
```diff
@@ -7,5 +7,5 @@ export default function dateFormatter(date?: string | null | Date | Moment, shor
 
 	if (short) return moment(date).format('MM/DD/YYYY');
 
-	return moment(date).format('MMMM Do, YYYY');
```
Comment (jlhogan) [inline src/Hedwig/ClientApp/src/utils/dateFormatter.ts:10]: why this change? was matching the copy from Ashley here (not arguing against this change, just interested)

File: src/Hedwig/ClientApp/src/components/Header/_index.scss (hunk 1 above)
Comment (mmazanec22) [reply src/Hedwig/ClientApp/src/components/Header/_index.scss:17]: There was a sass error-- I guess the latest version doesn't allow `@extend` with pseudo classes ¯\_(ツ)_/¯

File: src/Hedwig/ClientApp/src/containers/Reports/ReportDetail/ReportDetail.tsx (hunk 3 above)
Comment (mmazanec22) [reply src/Hedwig/ClientApp/src/containers/Reports/ReportDetail/ReportDetail.tsx:66]: Was trying to stay true to the latest invision prototype, where it's formatted like this

File: src/Hedwig/ClientApp/src/utils/dateFormatter.ts (hunk 4 above)
Comment (mmazanec22) [reply src/Hedwig/ClientApp/src/utils/dateFormatter.ts:10]: Latest invision has this format
